scripts/exregional_aqm_lbcs.sh

```diff
@@ -84,11 +84,21 @@ yyyymmdd=${CDATE_MOD:0:8}
 mm="${CDATE_MOD:4:2}"
 hh="${CDATE_MOD:8:2}"
 
-if [ ${#FCST_LEN_CYCL[@]} -gt 1 ]; then
-  cyc_mod=$(( ${cyc} - ${DATE_FIRST_CYCL:8:2} ))
-  CYCLE_IDX=$(( ${cyc_mod} / ${INCR_CYCL_FREQ} ))
-  FCST_LEN_HRS=${FCST_LEN_CYCL[$CYCLE_IDX]}
-fi
+case $cyc in
+  00 )
```

Comment:
@lgannoaa, this case-statement is too case-specific. The original one is more functional and simple.

Reply:
This assignment does not need completed array calculation and shell dependent coding.

Reply:
The production branch should be as close to the 'develop' branch as possible. Your change is too specific for the production case. Both the original one and your change will work for the production case. If so, I don't see any reasons for this change.

Reply:
@lgannoaa Is your objection just the more complicated shell syntax?  If the original version is working, and if there is a desire to keep this relatively close to the development SRW app, I'm not sure I see the need for this change